File: go/orm/internal/conformance/amendfixture/V0001.go
package amendfixture

import (
	"github.com/kosmassp/SimpleOrm/go/orm"
	amendwidget "github.com/kosmassp/SimpleOrm/go/orm/internal/conformance/amendfixture/Table/AmendWidget"
)

type V0001 struct{}

func (V0001) Compose(version *orm.VersionBuilder) {
	version.Apply(amendwidget.V0001_Create{})
}

File: php/tests/Conformance/AmendFixture/V0001.php
<?php

declare(strict_types=1);

namespace SimpleOrm\Tests\Conformance\AmendFixture;

use SimpleOrm\Migrations\MigrationVersion;
use SimpleOrm\Migrations\VersionBuilder;
use SimpleOrm\Tests\Conformance\AmendFixture\Table\AmendWidget\V0001_Create;

final class V0001 extends MigrationVersion
{
    public function compose(VersionBuilder $version): void
    {
        $version->apply(V0001_Create::class);
    }
}
